moto/efs/models.py

```diff
@@ -0,0 +1,542 @@
+"""Implement models for EFS resources.
+
+See AWS docs for details:
+https://docs.aws.amazon.com/efs/latest/ug/whatisefs.html
+"""
+
+from __future__ import unicode_literals
+
+import json
+import time
+from copy import deepcopy
+from hashlib import md5
+
+from boto3 import Session
+
+from moto.core import ACCOUNT_ID, BaseBackend, CloudFormationModel
+from moto.core.utils import (
+    camelcase_to_underscores,
+    get_random_hex,
+    underscores_to_camelcase,
+)
+from moto.ec2 import ec2_backends
```

Comment:
As of 2.x, Moto supports installation of only specific modules (`pip install moto[efs]`). At the moment that would fail, as EFS does not specify additional dependencies, but EC2 does - so importing any EC2 model will fail with a `module not found` exception.
As far as I can tell, we only need the EC2 reference in create_mount_target/delete_mount_target. Can we lower the import inside those methods?
That way people will be able to install EFS alone without needing any additional dependencies - only when calling those two methods will they receive the notification that EC2 dependencies are not installed.

Reply:
> Hi @pagreene, thanks for the PR! Looks good overall - just left a few comments.
> 
> Note that the linter is failing at the moment - you can use `make format` to fix this.
> Make sure that you have right version of black installed (different versions will produce different results): `black==19.10b0`

Looks like I just forgot to lint a file in cloudformation. That should be fixed now.

Reply:
> As of 2.x, Moto supports installation of only specific modules (`pip install moto[efs]`). At the moment that would fail, as EFS does not specify additional dependencies, but EC2 does - so importing any EC2 model will fail with a `module not found` exception.
> As far as I can tell, we only need the EC2 reference in create_mount_target/delete_mount_target. Can we lower the import inside those methods?
> That way people will be able to install EFS alone without needing any additional dependencies - only when calling those two methods will they receive the notification that EC2 dependencies are not installed.

Unfortunately it is also used (currently) whenever a FileSystem is created, specifically to look up the availability zone ID given an availability zone name. Is there a way to do this without using ec2's `describe_availability_zones`?

Reply:
Ah, right. Don't think there's another way to to this at at the moment, no.
Looking into to it a bit more, we'll need to refactor the dependencies soon anyway. So let's leave this as is for now - I'll find a better way of doing this later on 👍